### PR TITLE
add fonts, styles and skeleton

### PR DESCRIPTION
**Pull request by ProxN**
1. added google fonts to the project (Montserrat - Bitter)
2. browser styles reset
3. added global styles using` styled-components`
4. added routes HomePage ` '/'` and SearchPage` '/search'`
5. Folder structure (pages,components,styles)

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
-      content="Web site created using create-react-app"
+      content="Reddit timer is a tool for content creators."
```
Comment (jkettmann) [inline public/index.html:10]: Great. I totally forgot about those :-)

File: src/components/layout/layout.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import GlobalStyles from '../../styles/globalStyles';
+
+const Layout = ({ children }) => {
+  return (
+    <div>
+      <GlobalStyles />
+      <Link to="/">Home</Link>
```
Comment (jkettmann) [inline src/components/layout/layout.js]: Are these links for testing purposes? If so better remove them from the PT

File: src/styles/fonts.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
+import { css } from 'styled-components';
```
Comment (jkettmann) [inline src/styles/fonts.js]: You can import Google fonts inside the `<head>` tag or a CSS file. For the source URL you can go to https://fonts.google.com/ select the fonts and click the banner `2 families selected`. You can even customize your selection and load only certain dunt weights

File: src/styles/reset.js [hunk 4]
```diff
@@ -0,0 +1,197 @@
+import { css } from 'styled-components';
+
+export default css`
+  /* http://meyerweb.com/eric/tools/css/reset/
```
Comment (jkettmann) [inline src/styles/reset.js]: Fyi an alternative is normalize.css. you can install it via npm and simply import it somewhere in your js code

File: src/styles/globalStyles.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
+import { createGlobalStyle } from 'styled-components';
```
Comment (jkettmann) [inline src/styles/globalStyles.js]: styled-components is a great choice :-)

File: src/styles/globalStyles.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+        width:100%;
+    }
+    #root,
+    body{
+        width:100%;
+        height:100%;
+    }
+    body{
+        line-height:1.5;
+        background-color:${theme.color.grayLight};
```
Comment (jkettmann) [inline src/styles/globalStyles.js]: Can't you access the theme from the props like you usually would do? 

https://spectrum.chat/styled-components/general/i-cant-use-my-theme-in-createglobalstyle-function-styled-components-v4-react-v16-6-3~0978b404-ab71-45c9-8f75-0862abde4eb5

**Review by jkettmann — CHANGES_REQUESTED** [review]
Great job. Only had a couple of minor comments

File: src/styles/globalStyles.js (hunk 6 above)
Comment (ProxN) [reply src/styles/globalStyles.js]: I didn't know this, Thanks

File: public/index.html [hunk 7]
```diff
@@ -7,9 +7,18 @@
...
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link
```
Comment (jkettmann) [inline public/index.html:13]: You can even combine the two links into one. I guess you can separate the fonts with `|`. Not 100% sure though

**Review by jkettmann — CHANGES_REQUESTED** [review]
Another minor thing

File: public/index.html (hunk 7 above)
Comment (ProxN) [reply public/index.html:13]: yes it works

File: public/index.html [hunk 8]
```diff
@@ -11,13 +11,13 @@
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
     <link
-      href="https://fonts.googleapis.com/css?family=Montserrat:400,500,700&display=swap"
+      href="https://fonts.googleapis.com/css?family=Montserrat:400,500,700|Bitter:400,700"
       rel="stylesheet"
     />
-    <link
+    <!-- <link
```
Comment (jkettmann) [inline public/index.html]: Sorry, one last thing :-) It would be better to remove this line instead of commenting it out

**Review by jkettmann — CHANGES_REQUESTED** [review]

File: public/index.html (hunk 8 above)
Comment (ProxN) [reply public/index.html]: done.

**Review by jkettmann — APPROVED** [review]
:+1:

**Pull request merged**